server/commands/subscriptionCreator.test.ts

```diff
@@ -0,0 +1,103 @@
+import { sequelize } from "@server/database/sequelize";
+import { Subscription, Event } from "@server/models";
+import { buildDocument, buildUser } from "@server/test/factories";
+import { flushdb } from "@server/test/support";
+import subscriptionCreator from "./subscriptionCreator";
+
+beforeEach(() => flushdb());
+
+describe("subscriptionCreator", () => {
+  const ip = "127.0.0.1";
+  const enabled = true;
+  const subscribedEvent = "documents.update";
+
+  it("should create subscription", async () => {
+    const user = await buildUser();
+    const document = await buildDocument({
+      userId: user.id,
+      teamId: user.teamId,
+    });
+
+    const subscription = await sequelize.transaction(async (transaction) =>
+      subscriptionCreator({
+        user: user,
+        documentId: document.id,
+        event: subscribedEvent,
+        enabled,
+        ip,
+        transaction,
+      })
+    );
+
+    const event = await Event.findOne();
+
+    expect(subscription.documentId).toEqual(document.id);
+    expect(subscription.userId).toEqual(user.id);
+    expect(event?.name).toEqual("subscriptions.create");
+    expect(event?.modelId).toEqual(subscription.id);
+    expect(event?.actorId).toEqual(subscription.userId);
+    expect(event?.userId).toEqual(subscription.userId);
+    expect(event?.documentId).toEqual(subscription.documentId);
+  });
+
+  it("should not create anther subscription if one already exists", async () => {
```

Comment:
```suggestion
  it("should not create another subscription if one already exists", async () => {
```

Reply:
Might want to add a test for if the existing subscription is disabled, as it would be failing currently 😉